server/lib/buff_server/packages.ex

```diff
@@ -50,7 +53,10 @@ defmodule BuffServer.Packages do
       {:error, %Ecto.Changeset{}}
 
   """
-  def create_package(attrs \\ %{}) do
+  def create_package(attrs \\ %{}, artifact_binary) do
+    bucket_path = "/#{attrs.name}/artifact"
```

Comment:
I have a suspicion that this whole working with the package storage will get bigger, might be worth while introducing another context of PackageStorage that the Packages context talk to.(same as i did with Token)
But we can do it later.

Reply:
when it'll grow bigger I'll refactor it, bit of an overkill right now for 2 lines of code